bmxlib-cbmx/include/cbmx/writer.h

```diff
@@ -37,6 +37,7 @@ struct MxfConfig
     int aes3;
     int kag_512;
     int single_pass;
+    const char *timecode;
```

Comment:
is it possible to use syntax:
`const char* timecode`
for me the type is a pointer of char, it's not a timecode pointer